[debug] [2021-04-23T11:43:49.162Z] ----------------------------------------------------------------------
[debug] [2021-04-23T11:43:49.165Z] Command:       /usr/local/bin/firebase /home/ray/.cache/firebase/tools/lib/node_modules/firebase-tools/lib/bin/firebase emulators:start
[debug] [2021-04-23T11:43:49.165Z] CLI Version:   9.6.1
[debug] [2021-04-23T11:43:49.165Z] Platform:      linux
[debug] [2021-04-23T11:43:49.165Z] Node Version:  v12.18.1
[debug] [2021-04-23T11:43:49.171Z] Time:          Fri Apr 23 2021 20:43:49 GMT+0900 (Japan Standard Time)
[debug] [2021-04-23T11:43:49.171Z] ----------------------------------------------------------------------
[debug] [2021-04-23T11:43:49.171Z] 
[debug] [2021-04-23T11:43:49.184Z] >>> [apiv2][query] GET https://firebase-public.firebaseio.com/cli.json [none]
[debug] [2021-04-23T11:43:49.221Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2021-04-23T11:43:49.222Z] > authorizing via signed-in user
[info] i  emulators: Starting emulators: auth, functions, database, hosting {"metadata":{"emulator":{"name":"hub"},"message":"Starting emulators: auth, functions, database, hosting"}}
[warn] ⚠  emulators: It seems that you are running multiple instances of the emulator suite for project qr-lost-and-found. This may result in unexpected behavior. 
[debug] [2021-04-23T11:43:49.251Z] [hub] writing locator at /tmp/hub-qr-lost-and-found.json
[warn] ⚠  functions: The following emulators are not running, calls to these services from the Functions emulator will affect production: firestore, pubsub {"metadata":{"emulator":{"name":"functions"},"message":"The following emulators are not running, calls to these services from the Functions emulator will affect production: \u001b[1mfirestore, pubsub\u001b[22m"}}
[info] ✔  functions: Using node@12 from host. {"metadata":{"emulator":{"name":"functions"},"message":"Using node@12 from host."}}
[debug] [2021-04-23T11:43:49.321Z] defaultcredentials: writing to file /home/ray/.config/firebase/systemd_alert_gmail.com_application_default_credentials.json
[debug] [2021-04-23T11:43:49.352Z] Setting GAC to /home/ray/.config/firebase/systemd_alert_gmail.com_application_default_credentials.json {"metadata":{"emulator":{"name":"functions"},"message":"Setting GAC to /home/ray/.config/firebase/systemd_alert_gmail.com_application_default_credentials.json"}}
[debug] [2021-04-23T11:43:49.354Z] > refreshing access token with scopes: []
[debug] [2021-04-23T11:43:49.355Z] >>> HTTP REQUEST POST https://www.googleapis.com/oauth2/v3/token  
 <request body omitted>
[debug] [2021-04-23T11:43:49.503Z] <<< HTTP RESPONSE 200 {"date":"Fri, 23 Apr 2021 11:43:48 GMT","pragma":"no-cache","expires":"Mon, 01 Jan 1990 00:00:00 GMT","cache-control":"no-cache, no-store, max-age=0, must-revalidate","content-type":"application/json; charset=utf-8","vary":"X-Origin, Referer, Origin,Accept-Encoding","server":"scaffolding on HTTPServer2","x-xss-protection":"0","x-frame-options":"SAMEORIGIN","x-content-type-options":"nosniff","alt-svc":"h3-29=\":443\"; ma=2592000,h3-T051=\":443\"; ma=2592000,h3-Q050=\":443\"; ma=2592000,h3-Q046=\":443\"; ma=2592000,h3-Q043=\":443\"; ma=2592000,quic=\":443\"; ma=2592000; v=\"46,43\"","accept-ranges":"none","transfer-encoding":"chunked"}
[debug] [2021-04-23T11:43:49.521Z] >>> [apiv2][query] GET https://firebase.googleapis.com/v1beta1/projects/qr-lost-and-found/adminSdkConfig [none]
[debug] [2021-04-23T11:43:49.661Z] <<< [apiv2][status] GET https://firebase-public.firebaseio.com/cli.json 200
[debug] [2021-04-23T11:43:49.661Z] <<< [apiv2][body] GET https://firebase-public.firebaseio.com/cli.json {"cloudBuildErrorAfter":1594252800000,"cloudBuildWarnAfter":1590019200000,"defaultNode10After":1594252800000,"minVersion":"3.0.5","node8DeploysDisabledAfter":1613390400000,"node8RuntimeDisabledAfter":1615809600000,"node8WarnAfter":1600128000000}
